glsl/color.frag

```diff
@@ -1,6 +1,18 @@
 varying vec4 vColor;
 uniform bool EnableDepthPreview;
 
+float jet_r(float x) {
```

Comment:
Should braces go on a new line here as well (we do it in the rest of this file, too)?